app-misc/astrolog/astrolog-7.40-r1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-misc/astrolog-ephemeris
```

Comment:
☝️

Reply:
Done. Removed the patch from 7.40 ebuild and moved it into the 7.60 ebuild itself along with the version bumps.